examples/protocol_specification_ex/default.yaml

```diff
@@ -9,16 +10,16 @@ speech_acts:
   error:
     error_code: ct:ErrorCode
     error_msg: pt:str
-    error_data: pt:dict[pt:str, ct:ErrorData]
+    error_data: pt:dict[pt:str, pt:bytes]
+...
 ---
 ct:ErrorCode: |
   enum ErrorCodeEnum {
-      UNSUPPORTED_PROTOCOL = -10001;
-      DECODING_ERROR = -10002;
-      INVALID_MESSAGE = -10003;
-      UNSUPPORTED_SKILL = -10004;
-      INVALID_DIALOGUE = -10005;
+      UNSUPPORTED_PROTOCOL = 0;
```

Comment:
@5A11 negative values are not permitted and enums must start from 0 for protobuf 3

Reply:
1. This is a backwards-incompatible change
2.  we don't have to use the Protobuf `enum`. The value of the error code can be checked at runtime, so we have more flexibility on the values and we are implementation independent.

Reply:
Fair point. @5A11 thoughts?